medical_family/models/medical_family.py

```diff
@@ -20,7 +20,7 @@
 #
 ###############################################################################
```

Comment:
Please switch to shorter header styles - https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3